tests/TestCase/Consumption/LimitAttemptsExtensionTest.php

```diff
@@ -28,10 +30,7 @@ public function setUp(): void
         EventManager::instance()->setEventList(new EventList());
     }
 
-    /**
-     * @beforeClass
-     * @after
-     */
+    #[BeforeClass, After]
```

Comment:
Do we allow this, or should each attribute be in its own line?

Reply:
I think this is fine. We should allow simple forms for simple usage :smile: